tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -186,25 +187,35 @@ def test_brand_string(test_microvm_with_api, network_config):
     "0xc0000083",  # MSR_CSTAR
     "0xc0000100",  # MSR_FS_BASE
     "0xc0000101",  # MSR_GS_BASE
+    # MSRs below are required only on T2A however,
+    # we are adding them to the common exception list to keep things simple
+    "0x834"     ,  # LVT Performance Monitor Interrupt Register
+    "0xc0010007",  # MSR_K7_PERFCTR3
+    "0xc001020b",  # Performance Event Counter MSR_F15H_PERF_CTR5
+    "0xc0011029",  # MSR_F10H_DECFG also referred to as MSR_AMD64_DE_CFG
 ]
 # fmt: on
 
+# CPU templates which support MSR test
+MSR_SUPPORTED_TEMPLATES = ["T2S", "T2A"]
```

Comment:
nit: can order templates in alphabetical order

Reply:
done